### PR TITLE
Merge recent changes from rrthomas/ddclient

### PR DESCRIPTION
**Pull request by rrthomas**
Merge changes made in rrthomas/ddclient while ddclient/ddclient was archived.

### CODE REVIEW

**Comment by rrthomas** [comment]
@LenardHess Let me know if this PR looks OK, please!

**Comment by LenardHess** [comment]
Looks good to me, seeing as we looked over everything before in the other repo already.

Only thing that could use a second pair of eyes is my very [latest commit there](https://github.com/ddclient/ddclient/pull/551/commits/fcfd833dd0060dfdd1511dfa2592ec96df710ca1), if you wanna sanity check that

**Comment by rrthomas** [comment]
Latest commit looks OK, so merging this.

File: README.md [hunk 1]
```diff
@@ -75,7 +73,7 @@ through github.com. Please check out http://ddclient.net
 
 ## DOWNLOAD
 
-See https://github.com/ddclient/ddclient/releases
+See https://github.com/rrthomas/ddclient/releases
```
Comment (SuperSandro2000) [inline README.md:76]: that should probably be changed back

File: README.md [hunk 2]
```diff
@@ -1,70 +1,68 @@
...
+* [Sitelutions](https://www.sitelutions.com)
+* [woima.fi](https://woima.fi)
+* [Yandex](https://dns.yandex.com)
+* [Zoneedit](https://www.zoneedit.com)
+
+`ddclient` supports finding your IP address from many cable and DSL
+broadband routers.
+
+Comments, suggestions and requests: please file an issue at
+https://github.com/rrthomas/ddclient/issues/new
```
Comment (nemchik) [inline README.md:57]: https://github.com/ddclient/ddclient/pull/551/files#r1268541432

File: README.md (hunk 1 above)
Comment (rrthomas) [reply README.md:76]: Thanks, @SuperSandro2000, now fixed.

File: ddclient.in [hunk 3]
```diff
@@ -3,19 +3,12 @@
...
-# Support for multiple domain support for Namecheap by Robert Ian Hawdon 2010-09-03: https://robertianhawdon.me.uk/
-#
-# Initial Cloudflare support by Ian Pye, updated by Robert Ian Hawdon 2012-07-16
-# Further updates by Peter Roberts to support the new API 2013-09-26, 2014-06-22: http://blog.peter-r.co.uk/
+# Original Author: Paul Burry (paul+ddclient@burry.ca)
+# Current maintainers:
+#   Reuben Thomas <rrt@sc3d.org>
+#   Lenard Heß <lenard@rrhess.de>
 #
+# website: https://github.com/rrthomas/ddclient
```
[the remark this answers is not in the document]
Comment (rrthomas) [reply ddclient.in:11]: Thanks, @nemchik, now fixed.

**Pull request merged**